<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>项目符号(●)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(◎)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(◇)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(◆)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(1,2,3...)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(a,b,c...)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(A,B,C...)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(i,ii,iii...)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>项目符号(I,II,III...)</string>
     </property>
    </item>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>228</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="fontSizeComboBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1301</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="createAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAllAction"/>
    <addaction name="printAction"/>
    <addaction name="printPreviewAction"/>
    <addaction name="exitaction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="alignLeftAction"/>
     <addaction name="alignCenterAction"/>
     <addaction name="alignRightAction"/>
     <addaction name="alignJustifyAction"/>
    </widget>
    <widget class="QMenu" name="menu_D">
     <property name="title">
      <string>字体(&amp;D)</string>
     </property>
     <addaction name="boldAction"/>
     <addaction name="italicAction"/>
     <addaction name="underLineAction"/>
    </widget>
    <addaction name="menu_D"/>
    <addaction name="menu_2"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗体(&amp;W)</string>
    </property>
    <addaction name="closeAction"/>
    <addaction name="closeAllAction"/>
    <addaction name="tileAction"/>
    <addaction name="cascadeAction"/>
    <addaction name="nextAction"/>
    <addaction name="previousAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="createAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="saveAllAction"/>
   <addaction name="printAction"/>
   <addaction name="printPreviewAction"/>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="boldAction"/>
   <addaction name="italicAction"/>
   <addaction name="underLineAction"/>
   <addaction name="separator"/>
   <addaction name="alignLeftAction"/>
   <addaction name="alignCenterAction"/>
   <addaction name="alignRightAction"/>
   <addaction name="exitaction"/>
   <addaction name="colorAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="createAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_create.png</normaloff>:/images/file_create.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>创建一个新文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_open.png</normaloff>:/images/file_open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>打开一个文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_save.png</normaloff>:/images/file_save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAllAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_save_as.png</normaloff>:/images/file_save_as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="printAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_print.png</normaloff>:/images/file_print.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)...</string>
   </property>
  </action>
  <action name="printPreviewAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_print_preview.png</normaloff>:/images/file_print_preview.png</iconset>
   </property>
   <property name="text">
    <string>打印预览...</string>
   </property>
  </action>
  <action name="exitaction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_exit.png</normaloff>:/images/file_exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit_prev.png</normaloff>:/images/edit_prev.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit_next.png</normaloff>:/images/edit_next.png</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit_cut.png</normaloff>:/images/edit_cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit_copy.png</normaloff>:/images/edit_copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>颜色(&amp;C)..</string>
   </property>
  </action>
  <action name="boldAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/font_bold.png</normaloff>:/images/font_bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(&amp;B)</string>
   </property>
  </action>
  <action name="italicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/font_italic.png</normaloff>:/images/font_italic.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(&amp;I)</string>
   </property>
  </action>
  <action name="underLineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/font_underline.png</normaloff>:/images/font_underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线(&amp;U)</string>
   </property>
  </action>
  <action name="alignLeftAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/align_left.png</normaloff>:/images/align_left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐(&amp;L)</string>
   </property>
  </action>
  <action name="alignCenterAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/align_center.png</normaloff>:/images/align_center.png</iconset>
   </property>
   <property name="text">
    <string>居中(&amp;C)</string>
   </property>
  </action>
  <action name="alignRightAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/align_right.png</normaloff>:/images/align_right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐(&amp;R)</string>
   </property>
  </action>
  <action name="alignJustifyAction">
   <property name="text">
    <string>两端对齐(&amp;J)</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭(&amp;O)</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>关闭所有(&amp;A)</string>
   </property>
  </action>
  <action name="tileAction">
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="text">
    <string>层叠(&amp;C)</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>下一个(&amp;X)</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>前一个(&amp;V)</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
